### ISSUE TITLE
Broker websocket API doesn't like JSON strings with unicode escaping

### ISSUE THREAD

**Issue by simeonmiteff** [zeek/zeek#3989]
I encountered this while trying to send a string with HTML entities in it (e.g., `<ohai>`), and discovering that my JSON library (un?) helpfully escapes them as follows:

```json
{"@data-type":"string","data":"\u003cohai\u003e"}
```

I will patch https://github.com/corelight/go-zeek-broker-ws to turn off the HTML entity escaping feature and avoid this problem (I don't think we care about crappy browser security issues, and broker/CAF is happy to deserialise the string with HTML entities), but not handling escaped characters is probably a bug worth fixing.

**Comment by Neverlord** [comment]
> not handling escaped characters is probably a bug worth fixing

Absolutely. We actually already have someone working on this (https://github.com/actor-framework/actor-framework/pull/1949), so I'll take care that the fix finds its way to Broker/Zeek.

**Comment by simeonmiteff** [comment]
Much appreciated @Neverlord

**Comment by MP-Corelight** [comment]
@Neverlord, any chance this was resolved? I see that the PR you referenced above has merged.

**Comment by MP-Corelight** [comment]
Note: If this remains an open issue in Broker, we may recommend switching to the Zeek-based websocket implementation in v8.0 instead of continuing to work on the Broker websockets.
